app/__init__.py

```diff
@@ -1,13 +1,12 @@
-import re
-import os
+from __future__ import absolute_import
```

Comment:
Good spot, we don't. This was a hangover from when some of the logging was being developed in here.